build.sbt

```diff
@@ -2351,7 +2351,9 @@ lazy val `engine-runner` = project
           additionalCp = Seq(
             "runtime.jar",
             "runner.jar"
-          ),
+          ) ++ (`base-polyglot-root`
```

Comment:
Looks like this code is causing [sbt to fail to compile build script](https://github.com/enso-org/enso/actions/runs/8971799652/job/24641523363#step:5:20).

Reply:
it might have worked on an already built project because jars were already built. This however does not specify the dependency between subprojects.

Reply:
I see. Thank you 46b48d6da1 is the fix that prevents NPE.

Reply:
With 2556a2c we first `buildEngineDistribution` and only then compute the `additionalCp`. I doubt _passing a thunk_ instead of a value is the right `sbt` way of doing this, but it seems to do the trick.